packages/sampler/src/utils/propTables.js

```diff
@@ -5,6 +5,7 @@ const merge = (components, field) => {
 const mergeComponentMetadata = (displayName, ...components) => {
 	const fn = function () {};
 	fn.displayName = displayName;
+	fn.groupId = displayName;
```

Comment:
Is this supposed to be part of this PR?

Reply:
I described it in the description. It resolves a different, but equally important, knob issue with grouping.

Reply:
Ah, my bad.

:LGTM:

Reply:
Noticing an interesting side effect of this change for samples with multiple components (ProgressBar, Slider, IncrementSlider, and VideoPlayer) that each gets its own knob tab. For example, Slider's sample has a tab for Slider and one for SliderTooltip.

![image](https://user-images.githubusercontent.com/788456/53277170-6dc43100-36b7-11e9-8014-4ce40929cab7.png)


I'm not sure if this is a good thing or not but I'm okay with moving ahead with it unless someone objects.

Reply:
That's actually intentional. It was built with that in mind and has been present since it was added.

Here's a screenshot from 2.2.8:
<img width="526" alt="screen shot 2019-02-22 at 3 46 19 pm" src="https://user-images.githubusercontent.com/386529/53277506-11621100-36b9-11e9-9736-422199c16db7.png">

Reply:
:nice: Even better